hack/update-codegen.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -o errexit
+#set -o nounset
+#set -o pipefail
+
+
+SCRIPT_ROOT=$(git rev-parse --show-toplevel)
+
+# Grab code-generator version from go.sum.
+CODEGEN_VERSION=$(grep 'k8s.io/code-generator' go.sum | awk '{print $2}' | head -1)
```

Comment:
grab the code-generator from go.sum, so that if we change the version later, we don't need to update in multiple places like what we are doing now.